tests/ttnn/unit_tests/operations/eltwise/test_binary_maximum.py

```diff
@@ -52,8 +52,7 @@ def test_binary_max_int32(input_shapes, low_a, high_a, low_b, high_b, device):
     )
     tt_result = ttnn.maximum(tt_in_a, tt_in_b)
     output_tensor = ttnn.to_torch(tt_result)
-    pcc = ttnn.pearson_correlation_coefficient(golden, output_tensor)
-    assert pcc == 1
+    torch.equal(golden, output_tensor)
```

Comment:
I think you mean `assert torch.equal(golden, output_tensor)`. This just returns a `bool` that's ignored.